src/storage/v2/config.hpp

```diff
@@ -49,7 +49,7 @@ struct Config {
     uint64_t wal_file_flush_every_n_tx{100000};
 
     bool snapshot_on_exit{false};
-
+    bool restore_replicas_on_startup{false};
```

Comment:
```suggestion
    bool restore_replicas_on_startup{true};
```
should this default to true? in memgraph.cpp it is set to true. I don't know what our conventions are but this feels like an opportunity for misunderstandings if it sort of has two defaults, but the one in memgraph.cpp is "actually default" but bound later.

Reply:
The struct `Config::Durability` (where the parameter `restore_replicas_on_startup` is) is used at more places than `memgraph.cpp`, for instance in `main` from `mg_import.csv.cpp` at: https://github.com/memgraph/memgraph/blob/b737e53456e0bfb225ff9967b5379f9d3927beb8/src/mg_import_csv.cpp#L702-L708 
Idea was that I wanted these changes to be as transparent as possible excepted for `memgraph.cpp` where we explicitly say we want to restore replica(s) on startup.

Reply:
sounds good!